engine/flink/management/periodic/src/main/scala/pl/touk/nussknacker/engine/management/periodic/PeriodicProcessService.scala

```diff
@@ -160,6 +178,7 @@ class PeriodicProcessService(delegateDeploymentManager: DeploymentManager,
             }
           }
         case Left(error) =>
+          handleInvalidSchedule(deployment, error)
           // This case should not happen. It would mean periodic property was valid when scheduling a process
           // but was made invalid when rescheduling again.
           logger.error(s"Wrong periodic property, error: $error. Deactivating ${deployment.display}")
```

Comment:
This log is not needed here? It's already in handleInvalidSchedule?

Reply:
Yes, it's redundant

Reply:
done